plugins/uefi-capsule/fu-uefi-capsule-plugin.c

```diff
@@ -673,6 +674,13 @@ fu_uefi_capsule_plugin_get_default_esp(FuPlugin *plugin, GError **error)
 				}
 			}
 
+			/* in case an installer messes up the kind */
+			id_type = fu_volume_get_id_type(esp);
+			if (g_strcmp0(id_type, "btrfs") == 0) {
```

Comment:
I did wonder if we want an allow-list, e.g. vfat etc. A block-list is probably safer from a regression PoV.

Reply:
I think if we get another we should switch to allow-list.  Hopefully this is the last one.